core/const.py

```diff
@@ -70,3 +70,135 @@ class ProximityEnum(StrEnum):
     APPROACHING = "3. Approaching: next 6 months"
     CLOSE = "4. Close: next 3 months"
     IMMINENT = "5. Imminent: next month"
+
+
+# TLZ is given to any location outside the primary 12 ITL 1 regions as stated on the link below (previously NUTS & UKZ)
+# This introduces the problem that TLZ now maps to multiple locations (Channel Islands, Isle of Man, Non-geographic)
+# https://en.wikipedia.org/wiki/International_Territorial_Level
```

Comment:
@gidsg what do you think about this? Channel Islands and Isle of Man do not have their own ITL1 (previously NUTS1) codes. This means when we give them an 'other' code (UKZ), we can't then map it back to the actual location on the front-end. I imagine this is a problematic edge case? Or does this mean we just have 'Other'? Maybe a question for Maeve?

Reply:
Don't _think_ we'd fund projects in those regions so don't think this would concern us.

Reply:
ideal

Reply:
@cyrusdobbs can we reference ONS in the comment rather than Wikipedia? e.g. https://geoportal.statistics.gov.uk/documents/ons::local-authority-district-april-2021-to-lau1-to-itl3-to-itl2-to-itl1-january-2021-lookup-in-united-kingdom-1/about

Reply:
The reason I linked the Wikipedia page is because it was the only place I saw 'TLZ' mentioned as a code for anywhere outside of the main 12 codes. I tried to search around ONS but couldn't find it anywhere